add polls (#2732)
makerdao/community: governance/polls/Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024.md

discussion_link: https://forum.makerdao.com/t/sky-protocol-launch-season-token-and-product-launch-parameter-proposal/25031
parameters:
    input_format: single-choice
    victory_conditions:
        - { type : plurality }
    result_display: single-vote-breakdown
version: v2.0.0
options:
   0: Abstain
   1: Yes
   2: No
start_date: 2024-09-09T16:00:00
end_date: 2024-09-12T16:00:00
---
# Poll: Sky Protocol Launch Season - Token and Product Launch Parameter Proposal - September 9, 2024

The Governance Facilitators have placed a Governance Poll into the voting system on behalf of the Ecosystem team. This Governance [Poll](https://manual.makerdao.com/governance/governance-cycle/weekly-governance-cycle#weekly-governance-cycle-definitions-mip16c1) will be active for three days beginning on Monday, September 9 at 16:00 UTC.

**This is a binary vote.**
- **You may vote for a single option.**
- **You should vote for the option which you prefer.**
- **If you would accept either option, you should vote 'Abstain'.**

**This proposal will only take effect if the concurrent Atlas v2 Upgrade proposal passes.**

## Review

> The community can vote in this poll to express support or opposition to the following changes. In summary, this vovers the following items (find attached corresponding GitHub Repos, which include README files that explain all technical details):

Reviewer: ```suggestion
The community can vote in this poll to express support or opposition to the changes described below. In summary, this covers the following items (find attached corresponding GitHub repos, which include README files that explain all technical details):
```

> * New Token contracts and corresponding token converters

Reviewer: ```suggestion
* New Token contracts and corresponding token converters initializtion.
```

Reviewer: I think this list is generally a bit inconsistent in form. There are:

- verbal forms ("Use USDS to get SKY").
- nominalized forms ("SBE Upgrade")
- Some of the nominalized forms are 100% noun compounds, some are not ("The introduction of STR" in contrast to "STR introduction").

Additionally, list items are missing a period at the end and capitalization is inconsistent.

Author: In general we have tried to keep the wording as close to the post from BA Labs as possible. This is to ensure there is no ambiguity if the text differs between the two sources.

  * [USDS](https://github.com/makerdao/sdai/tree/susds) (Upgraded DAI)
  * [SKY](https://github.com/makerdao/sky) (Upgraded MKR)

> * The introduction of the [Sky Savings Rate (sUSDS)](https://github.com/makerdao/sdai/tree/snst)

Reviewer: ```suggestion
* The introduction of the [Sky Savings Rate (sUSDS)](https://github.com/makerdao/sdai/tree/susds)
```

Author: This is not a typo, we believe it is intentional to link to the snst branch as this is the branch and commit that was audited

* The introduction of [Sky Token Rewards (STR)](https://github.com/makerdao/endgame-toolkit/blob/master/README.md#farms)
  * Use USDS to get SKY
  * Use USDS to get 01 rewardsToken
* [GSM Delay](https://sky-atlas.powerhouse.io/#A.1.8.2_Governance_Security_Delay_Requirements-9349994e-9918-494c-9e9a-e993082ae055%7C0db30758) Changes
* Corresponding [Atlas V2](https://sky-atlas.powerhouse.io/) Active Data Updates

To learn more about Launch Season, refer to:

* [MakerDAO Endgame: Launch Season (final)](https://forum.makerdao.com/t/makerdao-endgame-launch-season-final/24740)
* [Sky has arrived!](https://forum.makerdao.com/t/sky-has-arrived/24959)
* [Atlas V2](https://forum.makerdao.com/t/atlas-v2-upgrade-poll-request/25010)
* [Endgame: Announcing the Token and Product Launch forum post](https://forum.makerdao.com/t/endgame-announcing-the-token-and-product-launch/25021/1)

### Launch Executive Proposal

* Set the Schedule Method time for the earliest spell execution: September 17, 12:00 UTC

### Deployment Requirements Before Token and Product Launch Spell

> * Tokens:

Reviewer: This list is inconsistent in a similar fashion.

I don't think the requirements are very clear. What does it mean that "Splitter" is required?

Author: See above comment

  * USDS
  * SKY
  * sUSDS
* DAI:USDS and MKR:SKY converters
* Create USDS/SKY UniV2 pool (if this does not exist already)
* FlapperUniV2 for the Smart Burn Engine
* Splitter
* SplitterMOM
* LitePSM USDS Wrapper
* USDSJoin
* SKY DssVestMintable
* SKY VestingRewardsDistribution
* Rewards distribution cron-job
* StakingRewards (rewardsToken: SKY, stakingToken: USDS)
* StakingRewards (rewardsToken: 01, stakingToken: USDS)

### Parameters

* New Tokens Initialization
  * [USDS](https://github.com/makerdao/nst)
  * [SKY](https://github.com/makerdao/ngt)
  * [Savings USDS (sUSDS)](https://github.com/makerdao/sdai/tree/susds)
    * Set Sky Savings Rate to **6.25%**
* Initialize token converters
  * DAI <> USDS converter

>     * DAI to USDS conversion (both ways). The exchange Rate is **1:1**

Reviewer 2: ```suggestion
    * DAI to USDS conversion (both ways). The exchange rate is **1:1**
```

  * MKR <> SKY converter
    * MKR to SKY conversion with an MKR/SKY conversion rate of **1:24,000**
* USDS/SKY UniV2 Pool Migration and Smart Burn Engine Upgrade to use New Tokens
  * USDS/SKY UniV2 Pool Migration
    * Migrate funds from DAI/MKR UniV2 Pool to USDS/SKY UniV2 Pool
  * Splitter Initialization:
    * `hump`: **55 million DAI**
    * `bump`: **65,000 DAI** (USDS in practice)
    * `burn`: **100%** (1.0 * WAD)
    * `hop`: **10,249 seconds**
  * Dss Flapper Initialization to Use New Tokens
    * `flapper`: FlapperUniV2
    * `pip`: OracleWrapper ([0x38e8c1D443f546Dc014D7756ec63116161CB7B25](https://etherscan.io/address/0x38e8c1D443f546Dc014D7756ec63116161CB7B25))
    * `want`: **0.98 * WAD**
* Sky Token Rewards (STR) Deployment
  * SKY (Supply USDS to get Sky Token Rewards in the form of SKY)
    * DssVest
      * Rewards Distribution Rate: **600,000,000 SKY**
      * Rewards Distribution Cap: **800,000,000 SKY**
      * Start Date relative to the spell cast time minus 7 days
        * `vestBgn`: `block.timestamp - 7 days`
      * End Date is set one year after Start Date defined by duration
        * `vestTau`: **365 days**

>       * Call `distrdibute()` in VestedRewardsDistribution contract in the spell execution

Reviewer 2: Should this be `distribute()` ?

Author: fixed in 3806c4d18ffbcb53765d683b0b79f0e3913c3c8a
